modules/Kconfig.tfm

```diff
@@ -51,6 +51,22 @@ config TFM_KEY_FILE_NS
 	  that should be used by the BL2 bootloader when signing non-secure
 	  firmware images.
 
+config TFM_PROFILE
+	string "The build profile used for TFM Secure image."
```

Comment:
not a "must" but I wonder if these options should have a default

Reply:
In TFM the default configuration is not a profile... so we should not add a default.

Reply:
ok!